### PR TITLE
Fixes #2039 - Default job values with 0 evaulate false

### LINKED ISSUE #2039: Create a job with IntegerVar and having the default set to 0 evaluates to False.
### Environment
* Python version: python3.9
* Nautobot version: 1.3.7

### Steps to Reproduce
1. create a new job with a variable definition of `IntegerVar(description="timer", default=0, required=True, min_value=0, max_value=3600)`
2. Load the job in the UI, default will be set to `None` and you must manually set 0.

### Expected Behavior
Default is used from the variable definition

### Observed Behavior
Using IntegerVar in conjunction with default=0 will evaulate to False and the default will not be displayed.

Fix could be:
`if default is not None:` 
here:
https://github.com/nautobot/nautobot/blob/69b691443cbeb8843aee1915b8956d003b9d994a/nautobot/extras/jobs.py#L611

### PR DESCRIPTION
**Pull request by jeffkala**
# Closes: #2039 
# What's Changed

Fixes the default job value to allow the IntegerVar to be a 0 and not evaluate to false.

### CODE REVIEW

File: nautobot/extras/tests/test_filters.py [hunk 1]
```diff
@@ -50,12 +59,11 @@
     Tag,
     Webhook,
 )
-from nautobot.ipam.models import IPAddress, VLAN
+from nautobot.ipam.models import VLAN, IPAddress
```
Comment (glennmatthews) [inline nautobot/extras/tests/test_filters.py:62]: Why this change?

**Review by glennmatthews — APPROVED** [review]

File: nautobot/extras/tests/test_filters.py (hunk 1 above)
Comment (jeffkala) [reply nautobot/extras/tests/test_filters.py:62]: Maybe I'm using a different Isort profile?  I have isort running on with the `black` profile. I've tried the `django` profile as well and it does even more.  I don't see anything in the styling guide for import sorting.  I can manually change it back if required.

Comment (jeffkala) [reply nautobot/extras/tests/test_filters.py:62]: from the isort docs as well.
```
NOTE: type here refers to the implied type from the import name capitalization. isort does not do type introspection for the imports. These "types" are simply: CONSTANT_VARIABLE, CamelCaseClass, variable_or_function
```

Comment (glennmatthews) [reply nautobot/extras/tests/test_filters.py:62]: Yeah, we don't have isort as part of the standard linters for this project at this time. Let's leave the import order alone for now to reduce churn where we're not otherwise making changes.

Comment (jeffkala) [reply nautobot/extras/tests/test_filters.py]: pushed the fix Had to add "--dont-order-by-type" to my isort
